akka-bench-jmh/src/main/scala/akka/remote/artery/SendQueueBenchmark.scala

```diff
@@ -53,7 +53,7 @@ class SendQueueBenchmark {
     val N = 100000
     val burstSize = 1000
 
-    val source = Source.queue[Int](1024, OverflowStrategy.dropBuffer)
```

Comment:
I see that the build fails in case of using a deprecated method in tests. I guess I need to migrate to the new Source.queue method everywhere. I'll do it later once we finalize the approach. Please, confirm it is the right way to deal with the problem (alternatively we may want to add some "ignore" statement, but I doubt it is the right way).

Reply:
You can use `@nowarn("msg=deprecated")` for those.